### PR TITLE
fix: set picture variable to point to avatarUrl

### LINKED ISSUE #55: AvatarURL present but default image is used
### Summary
I tried the new version of the github action, it worked really well, but for some reason only the default image is used
This is the result i got
image

I think the issue is from here.
https://github.com/andrii-bodnar/action-crowdin-contributors/blob/59a5c51e4067817afc2fe0605fa715cea5a298c8/src/contributors.ts#L134-L145

I tested this locally and the `crowdinMember.data` doesn't have the `avatarUrl`. But the `avatarUrl` is present in `user.user`.

I tried this and it worked
```typescript
if ('avatarUrl' in user.user) {
     picture = user.user.avatarUrl;
}
```

Is it okay to go with this?

### PR DESCRIPTION
**Pull request by RealRichi3**
### Summary
Default profile image is used even when the users crowdin profile is accessible

Fixes #55

### Changes 
- update `picture` variable to point to users crowdin profile image

### CODE REVIEW

**Comment by andrii-bodnar** [comment]
@RealRichi3 please run the `npm run build` and `npm run package` commands before committing changes

**Comment by RealRichi3** [comment]
alright

File: src/contributors.ts [hunk 1]
```diff
@@ -137,8 +137,8 @@ export class Contributors {
                     user.user.id
                 );
 
-                if ('avatarUrl' in crowdinMember.data && crowdinMember.data.avatarUrl) {
```
Comment (andrii-bodnar) [inline src/contributors.ts:140]: I'd suggest keeping this condition and trying to set `picture = user.user.avatarUrl` before the try-catch block if the `'avatarUrl' in user.user` exists

Comment (RealRichi3) [reply src/contributors.ts:140]: ohh, i'll update it now

Comment (RealRichi3) [reply src/contributors.ts]: done

Comment (andrii-bodnar) [reply src/contributors.ts]: @RealRichi3 the dist check WF still failing https://github.com/andrii-bodnar/action-crowdin-contributors/actions/runs/5313195576/jobs/9619077844?pr=56

could you please take a look?

Comment (andrii-bodnar) [reply src/contributors.ts]: Make sure you're running the same Node version locally as in the CI workflow - Node 16.x

**Review by andrii-bodnar — APPROVED** [review]

**Pull request merged**